test/images/agnhost/netexec/netexec.go

```diff
@@ -123,6 +127,7 @@ func init() {
 	CmdNetexec.Flags().IntVar(&udpPort, "udp-port", 8081, "UDP Listen Port")
 	CmdNetexec.Flags().IntVar(&sctpPort, "sctp-port", -1, "SCTP Listen Port")
 	CmdNetexec.Flags().StringVar(&httpOverride, "http-override", "", "Override the HTTP handler to always respond as if it were a GET with this path & params")
+	CmdNetexec.Flags().StringVar(&bindToIPs, "udp-bind-on", "", "A comma separated list of ip addresses to bind the udp server listeners to")
```

Comment:
naming is hard `udp-listen-addresses`, not attached to it, just want to hear other thoughts

Reply:
I don't have strong opinions, I found it quite hard to find the proper name :-)